iroha/src/sumeragi.rs

```diff
@@ -708,6 +540,197 @@ impl Debug for Sumeragi {
     }
 }
 
+/// Possible Peer's roles in consensus.
+#[derive(Eq, PartialEq, Debug, Hash)]
+pub enum Role {
+    /// Leader.
+    Leader,
+    /// Validating Peer.
+    ValidatingPeer,
+    /// Observing Peer.
+    ObservingPeer,
+    /// Proxy Tail.
+    ProxyTail,
+}
+
+/// Contains message structures for p2p communication during consensus.
+pub mod message {
+    use super::*;
```

Comment:
Maybe I missed something looking through - but do we need something from sumeragi here? 
If not - it's better to place normal imports here. That way we will see how much entities are connected in module and submodule.